packages/cli/src/oclif/commands/migrate.js

```diff
@@ -0,0 +1,121 @@
+const { flags } = require('@oclif/command');
+
+const BaseCommand = require('../ZapierBaseCommand');
+const PromoteCommand = require('./promote');
+const { callAPI, getLinkedApp } = require('../../utils/api');
+const { buildFlags } = require('../buildFlags');
+
+class MigrateCommand extends BaseCommand {
+  async perform() {
+    const percent = this.args.percent;
+    if (isNaN(percent) || percent < 1 || percent > 100) {
+      throw new Error('`PERCENT` must be a number between 1 and 100.');
```

Comment:
If you want, you could use `this.error(message)` instead of `throw new Error`. Their [custom error](https://github.com/oclif/errors/blob/b6903528d9d9a58a87347f3f86049393cc2e8d83/src/errors/cli.ts#L10-L51) doesn't do a ton, but it does clean the stack and do nice indenting and stuff. Same goes for all other thrown errors.